source/troubleshooting.rst

```diff
@@ -38,11 +38,11 @@ Cleaning out the Database
 -------------------------
 Our command to run a postgres container does implement docker volumes. Volumes allow our postgres data to be persistent, which means that the data will remain even if the postgres container is removed and recreated. If for any reason you would like to delete your postgres database, you will need to remove the docker volume.
 
-Removing the volume for RHEL 6 or Centos 6
+Removing the volume for RHEL 6 and Centos 6
```

Comment:
RHEL 6 or Centos 6
treat these as distinct things, not grouped things

Reply:
Done.